repos.d/deb/apertis.yaml

```diff
@@ -29,6 +29,15 @@
   repolinks:
     - desc: Apertis home
       url: https://www.apertis.org/
+    - desc: Apertis GitLab
+      url: https://gitlab.apertis.org/
+    - desc: Apertis Quality Assurance reports
+      url: https://qa.apertis.org/
+  packagelinks:
+    - type: PACKAGE_SOURCES
+      url: 'https://gitlab.apertis.org/pkg/{srcname}'
+    - type: PACKAGE_BUILD_STATUS
+      url: 'https://gitlab.apertis.org/pkg/{srcname}/-/pipelines?ref=apertis/v{{version}}'
```

Comment:
This seem to also refer to a branch name with suffix, so probably also `ref=apertis/{{v}}'`

Reply:
Fixed